### PR TITLE
(docs) Update README.md

### PR DESCRIPTION
**Pull request by vladdoster**
- fix grammar
- misc. cleanup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ You can find the full site at [perfectmediaserver.com](https://perfectmediaserve
...
+Perfect Media Server began life as a series of blog posts over at [blog.linuxserver.io](https://blog.linuxserver.io/tag/perfectmediaserver/). Those posts continue to be very popular, but a blog post can only get you so far. Therefore, I introduce perfectmediaserver.com - a wiki format information repository detailing all you need to know to build a free, open, and modular media server that will last for years to come.
 
-The primary technologies we recommend are [Linux](https://www.linux.org/), Containers (via [docker](https://www.docker.com/) and managed using [docker-compose](https://docs.docker.com/compose/)), [Proxmox](https://www.proxmox.com/en/), [MergerFS](https://github.com/trapexit/mergerfs/), [SnapRAID](http://www.snapraid.it/) and [ZFS](https://zfsonlinux.org/).
+The primary technologies we recommend are [GNU/Linux](https://www.linux.org/), Containers (via [docker](https://www.docker.com/) and managed using [docker-compose](https://docs.docker.com/compose/)), [Proxmox](https://www.proxmox.com/en/), [MergerFS](https://github.com/trapexit/mergerfs/), [SnapRAID](http://www.snapraid.it/) and [ZFS](https://zfsonlinux.org/).
```
Comment (RealOrangeOne) [inline README.md:9]: hmm https://www.reddit.com/r/interject/comments/bfyuzl/id_just_like_to_interject_for_a_moment/

Comment (vladdoster) [reply README.md:9]: 😎😎😎

Comment (RealOrangeOne) [reply README.md:9]: I was actually leaning towards not changing it at all.

Comment (vladdoster) [reply README.md]: Reverted.

Apologize to waste your time with that nit. I did it out of good faith and not to be a meme.

**Pull request merged**